src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/OpphørValidatorServiceTest.kt

```diff
@@ -0,0 +1,128 @@
+package no.nav.tilleggsstonader.sak.vedtak
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.tilleggsstonader.libs.utils.osloDateNow
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingType
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.ApiFeil
+import no.nav.tilleggsstonader.sak.util.saksbehandling
+import no.nav.tilleggsstonader.sak.util.tilSisteDagIMåneden
+import no.nav.tilleggsstonader.sak.util.vilkår
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.TilsynBarnTestUtil.vedtakBeregningsresultat
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.beregning.TilsynBarnBeregningService
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.domain.BeregningsresultatTilsynBarn
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.VilkårService
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.VilkårStatus
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.VilkårType
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkårsresultat
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.VilkårperiodeService
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.VilkårperiodeTestUtil
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.Vilkårperioder
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.felles.Vilkårstatus
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+
+class OpphørValidatorServiceTest {
+
+    private val vilkårperiodeService = mockk<VilkårperiodeService>()
+    private val vilkårService = mockk<VilkårService>()
+    private val tilsynBarnBeregningService = mockk<TilsynBarnBeregningService>()
+
+    val saksbehandling = saksbehandling(revurderFra = osloDateNow(), type = BehandlingType.REVURDERING)
+    val opphørValidatorService = OpphørValidatorService(vilkårperiodeService, vilkårService, tilsynBarnBeregningService)
+
+    @Test
+    fun kasterIkkeFeilVedKorrektData() {
```

Comment:
Pleier ikke testene å skrives sånn her? Vet ikke hva fordelene er ved det evt 🤷‍♀️ 
```suggestion
    fun `kaster ikke feil ved korrekt data`() {
```

Reply:
Det var lurt! a0004f1ba1f5f6f4db0745bfb69162a3e308d54c